apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -175,7 +175,8 @@ class PiskelEditor extends React.Component {
           if (this.props.currentAnimation !== key) {
             this.loadSelectedAnimation_(this.props);
           }
-        }
+        },
+        animationProps.frameCount
```

Comment:
Just curious - do you know where `frameCount` gets assigned?

Reply:
I'm not sure off the top of my head, we store it in the animation metadata but I don't know where the data comes from.